components/Stats.vue

```diff
@@ -292,15 +282,12 @@ import TimeLineChart from '@/components/TimeLineChart.vue'
 import TimeLineChartCountyComparison from '@/components/TimeLineChartCountyComparison.vue'
 import CasesSummary from '@/components/CasesSummary.vue'
 import HorizontalBarChart from '@/components/HorizontalBarChart'
-import Data from '@/data/data.json'
 import DataVTwo from '@/data/data.v2.json'
 import DataHospitalization from '@/data/data_hospitalization.json'
 import formatCountyData from '@/utils/formatCountyData'
 import formatCountyDataVTwo from '@/utils/formatCountyDataVTwo'
-import consolidateAllData from '@/utils/consolidateAllData'
 import formatCountyHospitalizationData from '@/utils/formatCountyHospitalizationData'
 import DataView from '@/components/DataView.vue'
-import countyColor from '@/static/data/countyColor.json'
```

Comment:
Moved to formatCountyData util but it might be better to have this live in the scss variables.